### PR TITLE
prints() and parenthesis for exceptions

### PR DESCRIPTION
**Pull request by BMaxV**
## Issue description
I was running radon cc and that threw error messages. Turns out there were still some python 2 remnants in there.

## Solution description
```
print "text" -> print("text")

...
except e1, e2 : > except (e1,e2):
...
```
## Checklist
I have done my best to ensure that…
* [x] …I have familiarized myself with the CONTRIBUTING.md file
* [x] …this change follows the coding style and design patterns of the codebase
* [x] …I own the intellectual property rights to this code
* [x] …the intent of this change is clearly explained
* [x] …existing uses of the Panda3D API are not broken
* [ ] …the changed code is adequately covered by the test suite, where possible.

### CODE REVIEW

File: panda/src/iphone/ipfreeze.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
 from direct.dist import FreezeTool
 
 def usage(code, msg = ''):
-    print >> sys.stderr, __doc__
-    print >> sys.stderr, msg
+    print( sys.stderr, __doc__)
+    print( sys.stderr, msg)
```
Comment (rdb) [inline panda/src/iphone/ipfreeze.py:21]: This is not the correct way to replace this syntax in Python 3, use `file=`

File: panda/src/iphone/ipfreeze.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def usage(code, msg = ''):
 
     try:
         opts, args = getopt.getopt(sys.argv[1:], 'h')
-    except getopt.error, msg:
+    except (getopt.error, msg):
```
Comment (rdb) [inline panda/src/iphone/ipfreeze.py:33]: This is not equivalent, use `as`.

File: panda/src/testbed/test_native_net2.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
 addr = SocketAddress()
 addr.setHost("127.0.0.1",6666)
-print addr.getIpPort()
+print (addr.getIpPort())
```
Comment (rdb) [inline panda/src/testbed/test_native_net2.py:18]: No space before parenthesis in function calls

**Review by rdb — CHANGES_REQUESTED** [review]

File: panda/src/iphone/ipfreeze.py [hunk 4]
```diff
@@ -17,8 +17,8 @@
 from direct.dist import FreezeTool
 
 def usage(code, msg = ''):
-    print( sys.stderr, __doc__)
-    print( sys.stderr, msg)
+    print(file=sys.stderr, __doc__)
+    print(file=sys.stderr, msg)
```
Comment (rdb) [inline panda/src/iphone/ipfreeze.py:21]:
```
>>> print(file=sys.stderr, "abc")
  File "<stdin>", line 1
SyntaxError: positional argument follows keyword argument
```

**Review by rdb — APPROVED** [review]

**Comment by BMaxV** [comment]
Changes were merged.

**Pull request closed without merging**